src/shared/containers/GetOrganizations.tsx

```diff
@@ -35,108 +35,95 @@ import {convertStringToEpoch} from 'src/shared/utils/dateTimeUtils'
 import {updateReportingContext} from 'src/cloud/utils/reporting'
 
 // Types
-import {Me} from 'src/client/unityRoutes'
 import {PROJECT_NAME} from 'src/flows'
 import {RemoteDataState} from 'src/types'
 
 // Thunks
 import {getQuartzIdentityThunk} from 'src/identity/actions/thunks'
 
-const canAccessCheckout = (me: Me): boolean => {
-  if (Boolean(me?.isRegionBeta)) {
-    return false
-  }
-  return me?.accountType !== 'pay_as_you_go' && me?.accountType !== 'contract'
-}
-
 const GetOrganizations: FunctionComponent = () => {
-  const {status, org} = useSelector(getAllOrgs)
-  const identity = useSelector(selectQuartzIdentity)
-  const quartzMeStatus = useSelector(getQuartzMeStatus)
-  const quartzMe = useSelector(getQuartzMe)
-  const quartzIdentityStatus = useSelector(selectQuartzIdentityStatus)
   const {id: meId = '', name: email = ''} = useSelector(getMe)
-  const {account} = identity.currentIdentity
+  const {account, user, org: quartzOrg} = useSelector(selectCurrentIdentity)
+  const identityLoadingStatus = useSelector(selectQuartzIdentityStatus)
+  const {status: orgLoadingStatus, org} = useSelector(getAllOrgs)
```

Comment:
`quartzOrg` is the user's current organization as retrieved from quartz (api/v2/quartz/identity) - see the identity reducer and src/identity/actions/thunks.

`org` - from selector `getAllOrgs` - is the user's current organization as retrieved from idpe (/api/v2/orgs) - see the organizations (and resources) reducer and src/organizations/actions/thunks.

In `CLOUD` the id and name of `quartzOrg` and `org` are the same and refer to the same org. In OSS, only `org` is populated (because quartz doesn't exist, and its endpoints aren't called), and `quartzOrg` is an object with default values (empty string). 

Depending on whether/how the app splits: in CLOUD, continuing to use the IDPE org id is fine and avoids us adding `if (cloud)` everywhere in the app that the org id is currently used. But there will no longer be any need going forward to refer to the IDPE org in CLOUD, as that endpoint isn't giving us anything that the new quartz endpoints don't provide. Whereas in OSS we will continue to need the IDPE org endpoint and reducer.

Reply:
>In `CLOUD` the id and name of `quartzOrg` and `org` are the same and refer to the same org. In OSS, only `org` is populated (because quartz doesn't exist, and its endpoints aren't called), and `quartzOrg` is an object with default values (empty string).

so in both cloud and oss environments, we're going to call an unnecessary selector? I think we should have a single `org` selector called `getOrg` that encapsulates and abstracts the logic of whether we're in cloud or oss environments so that at the component level, we only have to call `useSelector(getOrg)` and not worry about it. Because there will surely be similar questions about why we're selecting both `org` and `quartzOrg` from other people.

What do you think?

Reply:
I like that idea - and let me play with it in the current files here and see what can be improved.

Replacing this logic elsewhere is a bigger overhaul though - the idpe org selector is called all over the application, so trying to unify the app to use a single org selector is its own project.